sui_programmability/framework/sources/Math.move

```diff
@@ -21,4 +21,22 @@ module Sui::Math {
             y
         }
     }
+
+    /// Get Square Root for `x`
```

Comment:
I think tit would be useful for this to explain what happens when the result is fractional.